script/test

```diff
@@ -21,7 +21,7 @@ if [ "$FMT_ERRS" -gt "0" ]; then
 fi
 echo "ok  gofmt"
 
-VET_ERRS=$(go tool vet -test $pkgs 2>&1 | tee /dev/tty | wc -l)
```

Comment:
The true switch here should be `-all -shadow`
https://github.com/vsco/bricks/pull/75